app/javascript/modules/validate-forms.js

```diff
@@ -9,6 +9,7 @@
     Array.prototype.slice.call(forms)
       .forEach(function (form) {
         form.addEventListener('submit', function (event) {
+          if (event.submitter.id === 'save-draft-button') return // do not run client side validation for saving drafts
```

Comment:
Can this be moved to `app/javascript/controllers/edit_deposit_controller.js`?  This current file applies to all forms, but this functionality is only for a particular form.

Reply:
i was concerned about doing this globally too, which is why we try to do it this way (it works as expected except for buttons with an id of "save-draft-button").  not sure how we would extract just this scenario without removing the global event listener?

Reply:
Note: this is now used as well for the collection form too to bypass client side validation for collections saved as drafts.